packages/elasticsearch-plugin/src/indexing/indexer.controller.ts

```diff
@@ -46,9 +46,7 @@ import {
 
 import { createIndices, getClient, getIndexNameByAlias } from './indexing-utils';
 import { MutableRequestContext } from './mutable-request-context';
-
-const REINDEX_CHUNK_SIZE = 2500;
-const REINDEX_OPERATION_CHUNK_SIZE = 3000;
+import { asyncObservable } from "@vendure/core/src";
```

Comment:
This import is causing issues due to importing from `/src`. Add the `asyncObservable` import back into the group of `@vendure/core` imports and hopefully this resolves the failing tests.

Reply:
Yes, didn't notice the error, I already fixed